x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -173,6 +176,12 @@ export function WaterfallItem({
         <HttpStatusCode item={item} />
         <NameLabel item={item} />
         <Duration item={item} />
+        {errorCount > 0 && item.docType === 'transaction' ? (
```

Comment:
Can we move this up before `Duration`? My intention was to also only show `Duration` on row hover, but that probably wasn't clear from the issue description 😬We can implement that in another PR, if we don't want to mess with this right now.

![image](https://user-images.githubusercontent.com/4104278/54268673-0db90000-457c-11e9-928b-2e6820bf34dd.png)

Reply:
Ah, that looks great... when all the labels are aligned left. When the spans are further on the right, and the label information aligns on the right, the duration should show on the left, because it looks a little strange when we reserve the space for the duration to be shown. Not sure how easily achievable this is? I've tried to figure out how we align the labels, especially when the span is on on the right.

<img width="719" alt="Screenshot 2019-03-14 at 09 06 03" src="https://user-images.githubusercontent.com/4104278/54340974-ed02b000-4638-11e9-8590-8aa90d4bf0d7.png">